client/index.html

```diff
@@ -286,6 +286,13 @@ <h1 class="title">Settings</h1>
 	<script src="js/libs.min.js"></script>
 	<script src="js/shout.templates.js"></script>
 	<script src="js/shout.js"></script>
-
+    <script src="js/ifvisible.js"></script>
+    <script>
+        ifvisible.on("focus", function() { 
```

Comment:
No real need for this to be in a script tag here, is there? Couldn't it be in js/shout.js.